yjit_iface.c

```diff
@@ -581,6 +581,8 @@ get_stat_counters(rb_execution_context_t *ec, VALUE self)
             counter_reader++;
             name_reader = name_end;
         }
+        rb_hash_aset(hash, ID2SYM(rb_intern("binding_allocation_count")), LL2NUM(yjit_runtime_counters.binding_allocations));
+        rb_hash_aset(hash, ID2SYM(rb_intern("local_variable_set_count")), LL2NUM(yjit_runtime_counters.binding_set));
```

Comment:
Don't need these anymore. Automatically covered by the loop above.

Reply:
Ooops!  Thanks!